src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -353,17 +352,15 @@ const AddTaskModal = props => {
     if (props.level >= 1) {
       const classificationMother = props.tasks.taskItems.find(({ _id }) => _id === props.taskId)
         .classification;
-        if(classificationMother){
-          setClassification(classificationMother);
-        }
-    }
-    else {
+      if (classificationMother) {
```

Comment:
I suggest renaming the variable too. 
Please rename all classifications and its related variables to "categories"